src/tags/htmltag.js

```diff
@@ -240,4 +240,13 @@ function toAttribute(key, value) {
   }
 }
 
+function escapeQuotes(obj){
+  Object.keys(obj).forEach(key=>{
+    if (typeof  obj[key] === "string") {
```

Comment:
@maoznir , why not using Utils.isString function? 
https://github.com/cloudinary/cloudinary_js/blob/7b8492869f9e3cef1b09f1f16437796beca0f580/src/util/jquery.js#L111

Reply:
done